concepticondata/conceptlists/Huang-1992-1820.tsv

```diff
@@ -518,7 +518,7 @@ Huang-1992-1820-516	516	door (front) / door (entrance)	大门			n	door (front) /
 Huang-1992-1820-517	517	window	窗子	1375	WINDOW	n	window
 Huang-1992-1820-518	518	beam (of building)	梁	1132	BEAM	n	beam (of building)
 Huang-1992-1820-519	519	rafter (of building)	椽子	829	RAFTER	n	rafter (of building)
-Huang-1992-1820-520	520	flight of steps	台阶	1158	STEP	n	flight of steps
+Huang-1992-1820-520	520	flight of steps	台阶			n	flight of steps
```

Comment:
again, the same concept in all lists. If we remove it, we need a new concept set, but I would say: it is STEP according to our current definition (we might be more precise: Step (of stairs)

Reply:
Yes, confirmed, the data from STEDT is thus either deliberately broken down to just a few words, or deliberately corrupted. I'll file an issue for [halenepal](https://github.com/lexibank/halenepal)...